hippod/store_container.py

```diff
@@ -47,6 +51,10 @@ def create_subcontainer_data_merge_issue_new(app, sha_major, sha_minor, object_i
     user_data = hippod.users.get(app, object_item['submitter'], 'submitter')
     d_sub['submitter'] = user_data[0]['fullname']
     d_sub['achievements'] = []
+    if anchor:
```

Comment:
the next lines should be done in a separate function. See my pre-pre-previous argument about scoping of things.